.env.dist

```diff
@@ -0,0 +1,27 @@
+APP_LABEL=Media Cloud AI
+APP_LOGO=SubTil_logo_preview.png
+APP_COMPANY_LOGO=logo_id_2018_fushia13_2lignes.png
+APP_COMPANY=FranceTélévisions
+APP_IDENTIFIER=subtil
+APP_DNS=backend.media-cloud.ai
+
+DATABASE_NAME=media_cloud_ai
+DATABASE_PORT=5432
+DATABASE_PASSWORD=postgres
+DATABASE_USERNAME=postgres
+DATABASE_HOSTNAME=database
+
+AMQP_HOSTNAME=rabbitmq
+AMQP_PORT=5672
+AMQP_MANAGEMENT_PORT=15672
+AMQP_USERNAME=mediacloudai
+AMQP_PASSWORD=mediacloudai
+AMQP_VHOST=media_cloud_ai_dev
+AMQP_TLS=false
+
+SSL=false
+PORT=8080
+
+ROOT_PASSWORD=mediacloudai
+ROOT_EMAIL=dev@media-cloud.ai
+HOSTNAME=backend
```

Comment:
it's not a really pertinent domain name

Reply:
It's not a big deal, but it's true that it will avoid confusion.